tenders-api/update_v6.sql

```diff
@@ -3,17 +3,13 @@
 Tenders DB Update script: v6 (project_user_mapping table)
 */
 CREATE TABLE IF NOT EXISTS project_user_mapping (
-    project_id                   INTEGER     NOT NULL,
+    project_user_mapping_id      SERIAL PRIMARY KEY,
     event_id                     INTEGER NOT NULL,
-    event_type                   VARCHAR(20) NOT NULL,
-    user_id                   VARCHAR(20) NOT NULL,
+    user_id                      VARCHAR(20) NOT NULL,
+    created_by                   VARCHAR(50) NOT NULL,
     created_at                   TIMESTAMP   NOT NULL,
-    updated_at                   TIMESTAMP,
-    PRIMARY KEY (project_id,event_id)  );
-
-ALTER TABLE project_user_mapping
-ADD CONSTRAINT project_user_mapping_project_fk FOREIGN KEY (project_id)
-    REFERENCES procurement_projects(project_id);
+    updated_by                   VARCHAR(50) ,
+    updated_at                   TIMESTAMP );
 
 ALTER TABLE project_user_mapping
 ADD CONSTRAINT project_user_mapping_event_fk FOREIGN KEY (event_id)
```

Comment:
just missing a semicolon

Reply:
not sure, where is this .look at the file and looks good. https://github.com/Crown-Commercial-Service/ccs-scale-db-scripts/blob/feature/SCAT-4251-user-mapping/tenders-api/update_v6.sql

Reply:
Ah sorry - my bad - must have just been looking at the diff 👍